src/bulk/common.ts

```diff
@@ -11,6 +11,10 @@ import {
   hasAtomicOperators,
   maxWireVersion
 } from '../utils';
+import executeOperation = require('../operations/execute_operation');
```

Comment:
Is it worth updating the export format for this while we're here?
This applies to some other imports as well but maybe we want to save this for a TS improvement work later.

Reply:
yeah I'd prefer to leave that to ts-specific work